WordPress/Classes/ViewRelated/Media/Tenor/TenorClient.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+typealias TenorResponseBlock = (Swift.Result<TenorResponse, Error>) -> ()
+
+enum TenorError: Error {
+    case networkError
+    case wrongDataFormat
+    case wrongUrl
+}
+
+class TenorClient {
```

Comment:
Since we just created this client this is a good choice for some fresh unit tests! We have OHHTTPStubs, what do you think of adding some tests to check the behavior of this client? In the cases of:

- Wrong/unavailable response
- Correct response
- Malformed URL